de.metas.business.rest-api/src/main/java/de/metas/rest_api/ordercandidates/request/JsonOLCandCreateRequest.java

```diff
@@ -50,29 +52,29 @@
 @Value
 public final class JsonOLCandCreateRequest
 {
-	JsonOrganization org;
+	private JsonOrganization org;
 
 	@ApiModelProperty( //
 			allowEmptyValue = false, //
 			value = "This translates to <code>C_OLCand.externalLineId</code>.\n"
 					+ "<code>externalLineId</code> and <code>dataSourceInternalName</code> together need to be unique.")
-	String externalLineId;
+	private String externalLineId;
```

Comment:
pls remove the private everywhere (`private final` is already implied by `@Value`)

Reply:
done